defos/src/defos_mac.mm

```diff
@@ -93,6 +93,20 @@ void defos_set_window_title(const char* title_lua) {
     [window setTitle:title];
 }
 
+void defos_set_window_icon(const char *icon_path)
+{
+    NSString *path = [NSString stringWithUTF8String:icon_path];
+    NSImage* image = [[NSImage alloc] initWithContentsOfFile: path];
+    [window setRepresentedURL:[NSURL URLWithString:path]];
+    [[window standardWindowButton:NSWindowDocumentIconButton] setImage:image];
+}
+
+char const* defos_get_bundle_root() {
+    NSString *bundlePath = [[NSBundle mainBundle] bundlePath];
+    const char *bundlePath_lua = [bundlePath UTF8String];
+    return bundlePath_lua;
```

Comment:
This is fine. It won't leak since it's owned by the bundle, but if we add a `free()` in `get_bundle_root()` as I suggested for HTML and Windows, we should copy this string into some memory we own.

Reply:
I wanted to do this (I asked you aboutt free()), it was my fault with conts, thx!

Reply:
done